src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -122,4 +135,16 @@ private function generateProxyClass(Definition $definition)
 
         return $generatedClass;
     }
+
+    /**
+     * Generate unique suffix for a definition.
```

Comment:
I would remove this docblock, it adds little value IMHO

Reply:
Done